.k8s/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Chart.Name }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  revisionHistoryLimit: 1
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+        release: {{ .Release.Name }}
+        repository: {{ .Release.Name }}-local
+        imageTag: latest
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "lynx-local:latest"
+          command: ["make", "serve", "PORT=80"]
+          volumeMounts:
+            - name: src
+              mountPath: /usr/src/app
+          imagePullPolicy: IfNotPresent
+          env: []
+          ports:
+            - name: http
+              containerPort: 80
```

Comment:
All the k8s ports are hardcoded, but unlike the `Makefile`, I'm not sure it's worth paramaterizing these for now